### PR TITLE
index: make navbar responsive

### PR DESCRIPTION
**Pull request by AdityaSrivast**
Fixes: #386 

Expected Behaviour: Currently the navbar is not responsive. But, it should be responsive.

Deployment Link: https://adityasrivast.github.io/fossasia.org/

### CODE REVIEW

**Review by samagragupta — CHANGES_REQUESTED** [review]
Please resolve the conflicts.

**Comment by AdityaSrivast** [comment]
@samagragupta I have made the changes. Can you please review again?

**Comment by rcya1** [comment]
@AdityaSrivast There are still more conflicts now that must be resolved.

File: css/custom.css [hunk 1]
```diff
@@ -101,6 +101,16 @@
...
   height: auto;
 }
+@media screen and (min-width: 769px) and (max-width: 991px) {
+  .nav-wrapper {
+    width: 100%;
+  }
+}
+@media screen and (min-width: 769px) and (max-width:800px) {
+  .nav-wrapper {
+    padding: 0px 3px;
```
Comment (aayusharora) [inline css/custom.css]: Remove px from 0

**Review by aayusharora — CHANGES_REQUESTED** [review]

**Comment by aayusharora** [comment]
@AdityaSrivast please check the suggestions

**Review by sarnava1 — CHANGES_REQUESTED** [review]
@AdityaSrivast please resolve the conflicts

**Comment by AdityaSrivast** [comment]
@aayusharora @samagragupta Can you please review again? Also merge if in order. This is the second time I am updating this PR for merge conflicts :sweat_smile:

**Review by samagragupta — CHANGES_REQUESTED** [review]
navbar
There is problem in nav-bar at `1199px x 754px`

**Comment by AdityaSrivast** [comment]
image
@samagragupta it's working fine for me in chrome, firefox and opera. Which browser are you using?

**Review by samagragupta — COMMENTED** [review]
@sarnava1 @aayusharora What are your views?
navbar
I am using chrome.

**Review by jakubpawlina — CHANGES_REQUESTED** [review]
There is no way to enter the menu:
zrzut ekranu 136

**Comment by mariobehling** [comment]
Closing this issue as there is no follow up anymore.

**Pull request closed without merging**